usaspending_api/data/authoritative_agency_list.csv

```diff
@@ -560,7 +560,7 @@ CGAC AGENCY CODE,FPDS DEPARTMENT ID,AGENCY NAME,AGENCY ABBREVIATION,REGISTERED I
 033,3352,Smithsonian Institution,SI,YES*,NO,OMB File,3352,John F. Kennedy Center for the Performing Arts,KENNEDY CENTER,John F. Kennedy Center for the Performing Arts,03,3301,John F. Kennedy Center for the Performing Arts,,,,J. F. KENNEDY CENTER FOR THE PERFORMING ARTS,J. F. Kennedy Center for the Performing Arts,TRUE,,http://www.kennedy-center.org/,
 033,3355,Smithsonian Institution,SI,YES*,NO,OMB File,3355,National Gallery of Art,,National Gallery of Art,02,3302,National Gallery of Art,,,,NATIONAL GALLERY OF ART,,TRUE,,,
 034,3400,International Trade Commission,USITC,YES,NO,OMB File,3400,International Trade Commission,USITC,,,3400,International Trade Commission,,,,,,FALSE,"Consistent with its statutory mandate, the Commission makes determinations in proceedings involving imports claimed to injure a domestic industry or violate U.S. intellectual property rights; provides independent tariff, trade and competitiveness-related analysis and information; and maintains the U.S. tariff schedule.",https://www.usitc.gov/,
-036,3600,Department of Veterans Affairs,VA,YES,YES,OMB File,3600,Department of Veterans Affairs,VA,DEPARTMENT OF VETERANS AFFAIRS,00,3600,Department of Veterans Affairs,GPO Acronym List,,YES,"VETERANS AFFAIRS, DEPARTMENT OF","VETERANS AFFAIRS, DEPARTMENT OF",FALSE,"To fulfill President Lincoln's promise "To fulfill President Lincoln's promise ""To care for him who shall have borne the battle, and for his widow, and his orphan"" by serving and honoring the men and women who are America's veterans.",https://www.va.gov/,DOVA.jpg
+036,3600,Department of Veterans Affairs,VA,YES,YES,OMB File,3600,Department of Veterans Affairs,VA,DEPARTMENT OF VETERANS AFFAIRS,00,3600,Department of Veterans Affairs,GPO Acronym List,,YES,"VETERANS AFFAIRS, DEPARTMENT OF","VETERANS AFFAIRS, DEPARTMENT OF",FALSE,"To fulfill President Lincoln's promise ""To care for him who shall have borne the battle, and for his widow, and his orphan"" by serving and honoring the men and women who are America's veterans.",https://www.va.gov/,DOVA.jpg

```

Comment:
Looks like the actual content within the mission statement has changed. The latest content should be: 
```
To fulfill President Lincoln's promise "To care for him who shall have borne the battle, and for his widow, and his orphan" by serving and honoring the men and women who are America's veterans.
```

Reply:
wait, isnt that the exact same?

Reply:
disregard, i see the duplicated content within the quotes. looks good.